packages/manager/apps/public-cloud/src/components/walkMe/walkme.less

```diff
@@ -0,0 +1,87 @@
+@import "~ovh-ui-kit/packages/oui-color/_variables";
+@import '~ovh-ui-kit/packages/oui-responsive/responsive';
+@import '~ovh-ui-kit/packages/oui-typography/_constants.less';
+@import '../../assets/theme/_variables.less';
+
+.tour-public-cloud-walkme {
+  pointer-events: auto;
+  max-width: 305px;
+
+  .oui-icon::before {
+    font-size: 1rem;
+  }
+
+  .oui-font-semibold {
```

Comment:
`oui-font-semibold` doesn't already use `@oui-font-semibold` ? Or it doesn't exist ?

Reply:
Nope it is only a variable not a class